plugin/ensime.vim

```diff
@@ -61,8 +61,10 @@ augroup ensime
     autocmd VimLeave *.java,*.scala call ensime#au_vim_leave(expand("<afile>"))
     autocmd VimEnter *.java,*.scala call ensime#au_vim_enter(expand("<afile>"))
     autocmd BufLeave *.java,*.scala call ensime#au_buf_leave(expand("<afile>"))
-    autocmd CursorHold *.java,*.scala call ensime#au_cursor_hold(expand("<afile>"))
-    autocmd CursorMoved *.java,*.scala call ensime#au_cursor_moved(expand("<afile>"))
+    if !has('timers'):
```

Comment:
I didn't find a way to conditionally define autocmd using Neovim's library. More precisely, `neovim.autocmd` decorator is used when the class (`NeovimEnsime` here) is loaded. You can't call it after the fact to define an autocmd.
A possibility would be to declare the CursorHold and CursorMoved autocmd in `ensime.vim` for Neovim and not in Python.